include/rocksdb/utilities/backupable_db.h

```diff
@@ -165,6 +165,33 @@ struct BackupableDBOptions {
   // db_session_id as a fallback.
   BackupTableNameOption share_files_with_checksum_naming;
 
+  // Option for custom checksum functions.
```

Comment:
Nit: "customized" instead of "custom"?

Reply:
I'd prefer "custom". As English is not our first language, maybe Peter or Andrew can help us resolve the conflict? XD